docker/src/main/java/org/jclouds/docker/compute/strategy/DockerComputeServiceAdapter.java

```diff
@@ -97,38 +97,34 @@ public DockerComputeServiceAdapter(DockerApi api) {
               .image(imageId)
               .exposedPorts(exposedPorts);
 
-      if (templateOptions.getCommands().isPresent()) {
-         containerConfigBuilder.cmd(templateOptions.getCommands().get());
+      if (!templateOptions.getCommands().isEmpty()) {
+         containerConfigBuilder.cmd(templateOptions.getCommands());
       }
 
-      if (templateOptions.getMemory().isPresent()) {
-         containerConfigBuilder.memory(templateOptions.getMemory().get());
+      if (templateOptions.getMemory() != null) {
```

Comment:
Simplify this by removing the null check conditionals? These methods accept null values, so let's call them directly.